dependencies.yaml

```diff
@@ -415,11 +444,19 @@ dependencies:
           - pytest-xdist
           - seaborn
           - *scikit_learn
-          - &xgboost xgboost>=2.1.0
           - statsmodels
           - tenacity
           - umap-learn==0.5.7
           - pynndescent
+  test_python_xgboost:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          # We must separate xgboost into its own list so that it is not
+          # included in the "all" key. The libxgboost package depends on librmm
+          # but we do not want to have a package depending on librmm in
+          # devcontainers since it should be built from source.
```

Comment:
I think that comment is slightly incorrect.
```suggestion
          # We must separate xgboost into its own list so that it is not
          # included in the "devcontainers" key. The libxgboost package depends on librmm
          # but we do not want to have a package depending on librmm in
          # devcontainers since it should be built from source.
```

Reply:
Fixed! I re-wrapped the comment in 4c8f7b663.